assets/app/views/browse/build-config.html

```diff
@@ -22,7 +22,7 @@
                 <button
                     class="btn btn-default hidden-xs"
                     ng-click="startBuild()"
-                    ng-disabled="!canBuild">
+                    ng-disabled="!canBuild()">
```

Comment:
how much is the function getting called

On Thu, Feb 25, 2016 at 12:47 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/browse/build-config.html
> <https://github.com/openshift/origin/pull/7605#discussion_r54134399>:
>
> > @@ -22,7 +22,7 @@
> >                  <button
> >                      class="btn btn-default hidden-xs"
> >                      ng-click="startBuild()"
> > -                    ng-disabled="!canBuild">
> > +                    ng-disabled="!canBuild()">
>
> Yeah, I made a similar comment to Jakub that might have led to the bug.
>
> The problem is it's 3 places in each controller you have to update the
> var, and if you forget one, the button is disabled. I thought it was too
> easy to get wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7605/files#r54134399>.
>